utils/font.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+import codecs
+from pathlib import Path
+import sys
+import argparse
+import logging
+from PIL import Image
+import math
+
+logger = logging.getLogger('font.py')
+
+
+def cmd_create_font_png(non_ascii_chars_path: Path, src_png_path: Path,
+                        char_width: int, char_height: int, ipp_path: Path,
+                        dst_png_path: Path):
+    if not non_ascii_chars_path.exists():
+        logger.error('no %s found', non_ascii_chars_path.name)
+        return 1
+    if not src_png_path.exists():
+        logger.error('no %s found', src_png_path.name)
+        return 1
+
+    # not using with because I want to use charset outside of its scope
+    file = open(non_ascii_chars_path.resolve())
+    chars = file.read()
+    char_set = chars.split()
+    file.close
```

Comment:
1. not calling `file.close()`
2. and you can use with, as `with` does not introduce a new scope 😊 (in Python, indentation != scope)
```Python
with open(non_ascii_chars_path, 'r') as file:  # btw, no need to resolve, I believe
     char_set = file.read().split()
print(char_set)  # it works! :)